### ISSUE TITLE
UWP support

### ISSUE THREAD

**Issue by AuroraDysis** [fluentsprings/ExpressMapper#119]
I can install package successfully, but throw a runtime error:

```
Could not load file or assembly 'System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
```

**Comment by anisimovyuriy** [comment]
Dear @AuroraDysis ,

Are still experiencing this issue - isn't it dot net core issue?

Thanks,
Yuriy

**Comment by AuroraDysis** [comment]
Yes, only create a UWP application and install ExpressMapper, you will find this exception.
with Microsoft.NETCore.UniversalWindowsPlatform 5.2.2

**Comment by AuroraDysis** [comment]
Some may be useful
[rx.net project.json](https://github.com/Reactive-Extensions/Rx.NET/blob/master/Rx.NET/Source/System.Reactive.Core/project.json)
![image](https://cloud.githubusercontent.com/assets/13709836/20469556/b4e20d1a-afdb-11e6-9972-d74c29a9cf68.png)
